fab/operations/db.py

```diff
@@ -0,0 +1,54 @@
+import datetime
```

Comment:
going to start splitting out major operations like `db`, `services`, `staticfiles` etc into the `operations` module. everything in an operation module should not depend on the global state `env` and it should not define any `@task`s. at least that's the goal thus far